Tests/XCUITests/NavigationTest.swift

```diff
@@ -22,12 +22,12 @@ class NavigationTest: BaseTestCase {
         // Check the url placeholder text and that the back and forward buttons are disabled
         XCTAssert(urlPlaceholder == defaultValuePlaceholder)
         if iPad() {
```

Comment:
I think we could simplify the if statement - the only difference seems to be that for the iPad we tap the url field.